code/getData.py

```diff
@@ -0,0 +1,57 @@
+import tweepy
+from tweepy import OAuthHandler
+import csv
+import time
+
+#API Credentials
+consumer_key = 'GvyzAMpV6G5xsOCf4HtjaKO52'
+consumer_secret = 'VZ8Ra1Vp46D5Gq5Otf6XVCaLzAVmRPHGG4MLHXpKSO7zWZJFPO'
+access_token = '804568727685447680-Wuv4RjqRvkp1OoAXeIVJLokl4boO1Hl'
+access_secret = 'HBcA0FJ8YBbg9SXGLy8JhdF2PKpGjyUeIvgZRBWzAZurZ'
+
+auth = OAuthHandler(consumer_key, consumer_secret)
+auth.set_access_token(access_token, access_secret)
+
+api = tweepy.API(auth)
+
+# Open/create a file to write data to
+csvFile = open('result.csv','w')
```

Comment:
One question though! Aren't we collecting tweets? Why is it in `.csv` format? To my understanding (correct me if I'm wrong), your program takes a word, searches for tweets (one by one) and adds them to a file (which is a CSV (why?))